src/test/ui/rust-2018/uniform-paths/fn-local-enum.rs

```diff
@@ -0,0 +1,15 @@
+// compile-pass
+// edition:2018
+
+#![feature(uniform_paths)]
+
+fn main() {
+    enum E { A, B, C }
+
+    use E::*;
```

Comment:
@petrochenkov one question that arose in the lang team meeting is -- did you intend to stabilize these sorts of paths, where `E` refers to a fn-local item that is in scope, at the same time as other uniform paths? I wouldn't necessarily have a problem with that personally, but it's not something that's been discussed before.

Reply:
I have some reservations about imports referring to `macro_rules` and couple of other things, but I'm not aware of any issues with items in blocks, like `E` in this example.

When lang team is ready to stabilize, then it can be stabilized, I don't have a strong opinion.
Stabilization can be easily done in multiple steps if necessary.
(And this PR in particular doesn't stabilize anything.)

Reply:
(I personally think this behavior makes a whole lot of sense)